course_gather/settings/base.py

```diff
@@ -146,3 +134,8 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, '../static/')
+
```

Comment:
Since INTERNAL_IPs is specific to debug toolbar we should move this to dev and local_dev. Production settings does not need to inherit this information.

Reply:
[According to the docs](https://docs.djangoproject.com/en/3.0/ref/settings/#internal-ips), it can be used for more than just the debug toolbar. But since the debug toolbar is now enabled for all IPs (7d65f79) we could just remove it if you want.

Reply:
Yeah I saw the same debug() statement, but we don't use that at all really :shrug:

Reply:
Removed.